awx/ui/src/screens/Project/shared/ProjectSubForms/ArchiveSubForm.js

```diff
@@ -14,25 +14,7 @@ const ArchiveSubForm = ({
   scmUpdateOnLaunch,
 }) => (
   <>
-    <UrlFormField
-      tooltip={
-        <span>
-          {t`Example URLs for Remote Archive Source Control include:`}
-          <ul css={{ margin: '10px 0 10px 20px' }}>
-            <li>
-              <code>
-                https://github.com/username/project/archive/v0.0.1.tar.gz
-              </code>
-            </li>
-            <li>
-              <code>
-                https://github.com/username/project/archive/v0.0.2.zip
-              </code>
-            </li>
-          </ul>
-        </span>
-      }
-    />
+    <UrlFormField tooltip={ProjectHelpTextStrings().archiveUrl} />
```

Comment:
do we want to use the `const projectHelpStrings = ProjectHelpTextStrings();` convention here?

Reply:
alternatively, could just destructure out what's needed: `const { archiveUrl } = ProjectHelpText()`.

Reply:
These are good finds.  I've elected not to destructure the object because in some files (if we were to be consistent with this format and I think we should be) it would add a bunch of lines to the file and not really provide any clarity of make it easier to maintain/read